policyengine_us/variables/gov/states/va/tax/income/va_agi_individual.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class va_agi_individual(Variable):
```

Comment:
For consistency (also note https://github.com/PolicyEngine/policyengine-us/issues/3351)
```suggestion
class va_agi_indiv(Variable):
```